sympy/functions/elementary/piecewise.py

```diff
@@ -276,6 +277,18 @@ def eval(cls, *_args):
 
             current_cond.add(cond)
 
+            #   Does not add intervals which are already covered entirely by 
+            #   previous intervals (in terms of ordering of arguments)
+            #   Piecewise((0, t>=4), (0, t<3), (1, t<=2), (3, t<4))
+            #   >>> Piecewise((0, (t >= 4) | (t < 3)), (3, True))
+            from sympy import Intersection
+            cu_inter = cond.as_set()
+            inter = Intersection(existing_intervals, cu_inter)
+            if((cu_inter == inter)):
```

Comment:
```suggestion
            if cu_inter == inter:
```

Reply:
@sylee957  Hi thanks for your suggestion! I am a first time Open Source contributor, so my code might not be upto the standards, so you are free to edit the code to make sure it meets the standards of SymPy.

Reply:
@sylee957 Hi, my code quality test is failing since I made some changes live on GitHub, can you check where it might be failing (if you got time), (I'm a beginner to using GitHub so I'm facing such trivial problems)